core/vm/fvm/ubig128.hpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "common/error_text.hpp"
+#include "common/outcome.hpp"
+#include "primitives/big_int.hpp"
+
+namespace fc::vm::fvm {
+  using primitives::BigInt;
+
+  struct Ubig128 {
+    uint64_t high{};
+    uint64_t low{};
+
+    static outcome::result<Ubig128> fromBig(const BigInt &big) {
+      if (big.sign() < 0) {
+        return ERROR_TEXT("Ubig128::fromBig negative");
+      }
+      if (big.is_zero()) {
+        return Ubig128{};
+      }
+      if (msb(big) >= 128) {
+        return ERROR_TEXT("Ubig128::fromBig too big");
+      }
+      struct It {
```

Comment:
Name is not clear

Reply:
It's local struct.  
It's `It(erator)`.  
How should I rename it?